### ISSUE TITLE
Undefined offset &&  Exception: The query is poorly formatted JSON provided.

### ISSUE THREAD

**Issue by johannesjo** [giovanniramos/PDO4You#8]
With this json:

```
{"query": [
    {
        "table": "el",
        "values": {
            "birthday": null,
            "country": null,
            "first_name": "aaaa",
            "last_name": "aaa",
            "gender": null,
            "email": "aaa@googlemail.com",
            "invited_friends": null,
            "newsletter": false
        }
    }
]}
```

I get those error messages:

```
Notice: Undefined offset: 1 in E:\projekte\el\app\php\db\pdo4you\pdo4you\PDO4You.class.php on line 981
 Exception: The query is poorly formatted JSON provided.
#0 E:\projekte\el\app\php\db\pdo4you\pdo4you\PDO4You.class.php:754
752
753                try {
754                    $jarr = self::parseJSON($json);
755
756                    if ($type == 'insert') {
#1 E:\projekte\el\app\php\db\pdo4you\pdo4you\PDO4You.class.php:883
881    public static function insert($json, $use = null)
882    {
883        return self::executeQuery($json, 'insert', $use);
884    }
885
#2 E:\projekte\el\app\php\postData.php:69
67    $sInitDB = 'mysql:host=' . $dbCfg['host'] . ';dbname=' . $dbCfg['db_name'] . ';port=3306';
68    PDO4You::getInstance('mysql', $sInitDB, $dbCfg['usr'], $dbCfg['pwd']);
69    $result = PDO4You::insert(json_encode($postData));
70    print $result;
```

I also wonder, why the local paths are shown, as the page is opened via xampp.

**Comment by giovanniramos** [comment]
Thank you @johannesjo for reporting

I'll test your query as soon sanarei your question

> I also wonder, why the local paths are shown, as the page is opened via xampp.
> - The error messages are displayed by default via localhost

**Comment by giovanniramos** [comment]
Hello, update your class and try this:

```
    // SQL insertion in JSON format
    insert: [
        {
            table: "el",
            values: {
                "birthday": null,
                "country": null,
                "first_name": "aaaa",
                "last_name": "aaa",
                "gender": null,
                "email": "aaa@googlemail.com",
                "invited_friends": null,
                "newsletter": false
            }
        }
    ]
```

**Comment by giovanniramos** [comment]
Fixed in version 4.1.
